aten/CMakeLists.txt

```diff
@@ -1,12 +1,20 @@
-cmake_minimum_required(VERSION 3.0)
-set(CMAKE_MODULE_PATH
-  ${CMAKE_CURRENT_SOURCE_DIR}/cmake
+if (CAFFE2_CMAKE_BUILDING_WITH_MAIN_REPO)
+  if (NOT BUILD_ATEN)
+    return()
+  endif()
+else()
+  cmake_minimum_required(VERSION 3.0 FATAL_ERROR)
+  project(ATen CXX C)
+endif()
+
+# Find modules
+list(APPEND CMAKE_MODULE_PATH
+  ${CMAKE_CURRENT_SOURCE_DIR}/../cmake/Modules
```

Comment:
I think this line combined with the fact that PR #6699 nuked FindMKL.cmake in /cmake causes the MKL build to fail. I have an ongoing PR #7170 to address this problem.